downloads.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Downloads
+layout: home
+nav_order: 2
+---
+
+# Downloads
+{: .fs-8 .fw-700 .text-center }
+
+- Source code:
+    - PSPDEV: [Download](https://github.com/pspdev/pspdev/releases/tag/latest)
+
+- Docker image:
+    - PSPSDK: [Download](https://github.com/pspdev/pspsdk/pkgs/container/pspsdk)
```

Comment:
```suggestion
Downloading the latest version for your system [here](https://github.com/pspdev/pspdev/releases/tag/latest).

Alternatively, a Docker container is available [here](https://github.com/pspdev/pspdev/pkgs/container/pspdev).
```
This was using the wrong link for the docker container. It stating source code is also not quite right.